client/examples/output/build_basic_output.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example basic --release
+
+use iota_client::{
+    block::{
+        address::Address,
+        output::{
+            feature::MetadataFeature,
+            unlock_condition::{
+                AddressUnlockCondition, ExpirationUnlockCondition, StorageDepositReturnUnlockCondition,
+                TimelockUnlockCondition, UnlockCondition,
+            },
+            BasicOutputBuilder, Feature,
+        },
+    },
+    Client, Result,
+};
+
+/// In this example we will send basic outputs with different feature blocks
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production!
+    dotenv::dotenv().ok();
+
+    let node_url = std::env::var("NODE_URL").unwrap();
+
+    // Create a client instance.
+    let client = Client::builder().with_node(&node_url)?.finish()?;
+
+    let token_supply = client.get_token_supply().await?;
+
+    let address = Address::try_from_bech32("rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy")?.1;
+
+    let basic_output_builder = BasicOutputBuilder::new_with_amount(1_000_000)?
+        .add_unlock_condition(UnlockCondition::Address(AddressUnlockCondition::new(address)));
+
+    let outputs = vec![
+        // most simple output
+        basic_output_builder.clone().finish_output(token_supply)?,
+        // with metadata feature block
+        basic_output_builder
+            .clone()
+            .add_feature(Feature::Metadata(MetadataFeature::new(vec![13, 37])?))
```

Comment:
What about issuer and tag features, do you also want to add examples for them?

Reply:
Ideally, yes 🤔 
Should I add them in this PR?

Reply:
Whatever you prefer

Reply:
Then I will create another issue and PR so that we can merge this PR faster 🤔

Reply:
https://github.com/iotaledger/iota.rs/issues/1652